src/menu.c

```diff
@@ -91,12 +91,54 @@ static void pubkey_display_data_selector(unsigned int idx) {
   }
 }
 
+//////////////////////////////////////////////////////////////////////////////////////
+// Display mode submenu
+
+void display_mode_data_change(enum DisplayMode display_mode) {
+    uint8_t value;
+    switch (display_mode) {
+        case DisplayModeUser:
+        case DisplayModeExpert:
+            value = (uint8_t) display_mode;
+            nvm_write((void *)&N_storage.settings.display_mode, &value, sizeof(value));
+            break;
+    }
+    ui_idle();
+}
+
+const char* const display_mode_data_getter_values[] = {
+    "User",
+    "Expert",
+    "Back"
+};
+
+static const char* display_mode_data_getter(unsigned int idx) {
+    if (idx < ARRAYLEN(display_mode_data_getter_values)) {
+        return display_mode_data_getter_values[idx];
+    }
+    return NULL;
+}
+
+static void display_mode_data_selector(unsigned int idx) {
+    switch (idx) {
+        case 0:
+            display_mode_data_change(DisplayModeUser);
+            break;
+        case 1:
+            display_mode_data_change(DisplayModeExpert);
+            break;
```

Comment:
Wdyt about landing back on the higher menu after a selection, instead of back to the app home screen? Not just for this option, but for all the settings I mean.
When I started testing this PR, I needed to allow blind signing and switch the display mode, and it felt weird to have to drill down a second time to do that. Not a big deal, just realized it was unexpected UX.